### ISSUE TITLE
Lamma2 start a test server got error

### ISSUE THREAD

**Issue by anhdungitvn** [huggingface/candle#387]
### Description
Folow example to start a test server for llama2 got error.
```
# README.MD
For llama2, run the following command to retrieve the weight files and start a test server:

cd candle-wasm-examples/llama2-c
wget https://karpathy.ai/llama2c/model.bin
wget https://github.com/karpathy/llama2.c/raw/master/tokenizer.bin
trunk serve --release --public-url /candle-llama2/ --port 8081
And then browse to http://localhost:8081/candle-llama2.

```
### Expected behavior
Server start sucessfully.
### Current behavior
```
2023-08-10T08:44:41.267930Z  INFO 📦 starting build
2023-08-10T08:44:41.268424Z  INFO spawning asset pipelines
2023-08-10T08:44:41.268905Z ERROR ❌ error
error from HTML pipeline

Caused by:
    0: error getting canonical path for "/hosted/workspace/1_user/...../candle/candle-wasm-examples/llama2-c/tokenizer.json"
    1: No such file or directory (os error 2)
2023-08-10T08:44:41.269446Z  INFO 📡 serving static assets at -> /candle-llama2/
2023-08-10T08:44:41.269498Z  INFO 📡 server listening at http://127.0.0.1:8081
```
### Steps to reproduce
```
git clone https://github.com/huggingface/candle.git

cd candle

cd candle-wasm-examples/llama2-c
wget https://karpathy.ai/llama2c/model.bin
wget https://github.com/karpathy/llama2.c/raw/master/tokenizer.bin
trunk serve --release --public-url /candle-llama2/ --port 8081

```

### Diagnosis
- tokenizer.json not found
```
# https://github.com/huggingface/candle/blob/main/candle-wasm-examples/llama2-c/index.html
# line no. 7
 <link data-trunk rel="copy-file" href="tokenizer.json" />
```

**Comment by LaurentMazare** [comment]
Ah thanks for reporting this, I recently changed this example to use a full tokenizer (so that it's easy to add a prompt) but forgot to update the instructions. Anyway I just merged #389 that tweak the instructions, the relevant bit is as follows. Can you give it a spin and let us know if there are still some issues?
```bash
wget https://huggingface.co/spaces/lmz/candle-llama2/resolve/main/tokenizer.json
```

**Comment by EmilLindfors** [comment]
Hey guys, 
I had the same problem yesterday, but additionally another issue surfaced after i fixed the json issue:  

`Loading script from “http://127.0.0.1:8081/worker.js” with importScripts() was blocked because of a disallowed MIME type (“”).`

I added a <base href="/"> to the index.html and removed the --public-url /candle-llama2/ from the run script to serve at the base path instead, which fixed the problem. 

Fyi, Im on Ubuntu 22.04 and FF 116

**Comment by anhdungitvn** [comment]
> Ah thanks for reporting this, I recently changed this example to use a full tokenizer (so that it's easy to add a prompt) but forgot to update the instructions. Anyway I just merged #389 that tweak the instructions, the relevant bit is as follows. Can you give it a spin and let us know if there are still some issues?
> 
> ```shell
> wget https://huggingface.co/spaces/lmz/candle-llama2/resolve/main/tokenizer.json
> ```
Thanks @EmilLindfors 

The issue was solved by this!

No more issues in my case, works well and runs faster than I expected.

**Comment by anhdungitvn** [comment]
> Hey guys, I had the same problem yesterday, but additionally another issue surfaced after i fixed the json issue:
> 
> `Loading script from “http://127.0.0.1:8081/worker.js” with importScripts() was blocked because of a disallowed MIME type (“”).`
> 
> I added a to the index.html and removed the --public-url /candle-llama2/ from the run script to serve at the base path instead, which fixed the problem.
> 
> Fyi, Im on Ubuntu 22.04 and FF 116

Thanks @EmilLindfors for the information.

In my case, after tokenizer.json fixed, it works well with no extra issue.

Fyi, Im on u 20.04.1, ff 116.0.2, cargo 1.70.0, rustup 1.26.0, and rustc 1.70.0.

I'm going to close this issue since it has been resolved.
